src/bgw/job.c

```diff
@@ -606,9 +606,26 @@ ts_bgw_job_find_with_lock(int32 bgw_job_id, MemoryContext mctx, LOCKMODE tuple_l
 	{
 		TupleInfo *ti = ts_scan_iterator_tuple_info(&iterator);
 		job = bgw_job_from_tupleinfo(ti, sizeof(BgwJob));
-		Assert(num_found == 0);
-		num_found++;
+		jobs = lappend(jobs, job);
+	}
+
+	if (list_length(jobs) > 1)
+	{
+		ListCell *cell;
+		foreach (cell, jobs)
+		{
+			BgwJob *job = (BgwJob *) lfirst(cell);
+			ereport(LOG,
```

Comment:
I approved yesterday, but shouldn't this a WARNING message?

Reply:
Well... the current behavior does not long anything and takes the latest entry, so nothing is at risk of stopping to work, so not sure that a warning is appropriate. A notice might be more appropriate.